airbyte-integrations/connectors-performance/destination-harness/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM airbyte/integration-base-java:dev AS build
+WORKDIR /airbyte
+
+ENV APPLICATION destination-harness
+
+COPY build/distributions/${APPLICATION}*.tar ${APPLICATION}.tar
+
+RUN tar xf ${APPLICATION}.tar --strip-components=1 && rm -rf ${APPLICATION}.tar
+
+FROM airbyte/integration-base-java:dev
+ARG TARGETARCH
+WORKDIR /airbyte
+
+ENV APPLICATION destination-harness
+# Kubectl and socat are needed in order to create a pod similar to cloud orchestrator
+# It brings up in cluster the destination pod to measure, and communicates over socat
+RUN curl -O https://s3.us-west-2.amazonaws.com/amazon-eks/1.25.6/2023-01-30/bin/linux/${TARGETARCH}/kubectl
+RUN chmod +x ./kubectl
+RUN mv ./kubectl /bin/
+RUN yum install -y socat && yum clean all
+
+COPY --from=build /airbyte /airbyte
```

Comment:
Rodi, remind me again, why do we need this?

Reply:
"build" here is the first layer of the multi stage container - `AS build` on ln. 1.
On this stage we untar the contents of the build output destination-harness.tar into /airbyte
So this command brings whatever we had under that directory to this final stage of this docker image.
Sorry - it's hard to convey this in writing. 
Like trying to explain the plot of "Insomnia"…

Reply:
oh got it! this makes sense!